<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPcDaemon</class>
 <widget class="QMainWindow" name="TestPcDaemon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>589</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TestPcDaemon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" rowspan="2">
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>监听</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="pushButtonDisConnect">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditPort">
        <property name="minimumSize">
         <size>
          <width>108</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>5025</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxIP_List">
        <property name="minimumSize">
         <size>
          <width>148</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBoxClient">
        <property name="minimumSize">
         <size>
          <width>154</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>设备型号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDev">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>设备</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PX00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SX00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plx00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1003C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1010C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HCPLx00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EX00</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonScan">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>远程扫描</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_readTimeout">
        <property name="text">
         <string>Read超时</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>更新共享内存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButtonSwitchOutput">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>切换输出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="labelConnectState">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NotConnect&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>子卡:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_subCard">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>子卡</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PLx00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E300</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_card">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>1/2/3/4</string>
        </property>
        <property name="placeholderText">
         <string>子卡号(如：2/3/4)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="2" colspan="2">
     <widget class="QTextEdit" name="textEditShareMemory">
      <property name="minimumSize">
       <size>
        <width>136</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>接收区</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditRecv">
         <property name="minimumSize">
          <size>
           <width>619</width>
           <height>288</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: green;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButtonSend">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButtonClear">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QTextEdit" name="textEditSend">
        <property name="minimumSize">
         <size>
          <width>668</width>
          <height>156</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_defReply">
        <property name="text">
         <string>自动回复</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
